cfgov/unprocessed/css/forms.less

```diff
@@ -607,44 +398,346 @@
 }
 
 .form-l_col-1-2 {
-    .respond-to-min(1060px, {
+    .respond-to-min(@bp-med-min, {
         .grid_column(6);
     });
 }
 
 .form-l_col-1-3 {
-    .respond-to-min(1060px, {
+    .respond-to-min(@bp-med-min, {
         .grid_column(4);
     });
 }
 
 .form-l_col-2-3 {
-    .respond-to-min(1060px, {
+    .respond-to-min(@bp-med-min, {
         .grid_column(8);
     });
 }
 
+.form-l_col-1-4 {
+    .respond-to-min(@bp-med-min, {
+        .grid_column(3);
+    });
+}
+
 .form-l_col-1 {
-    .respond-to-min(1060px, {
+    .respond-to-min(@bp-med-min, {
```

Comment:
You should be able to wrap all the `form-l_col…` classes in `respond-to-min`, see https://github.com/cfpb/cfgov-refresh/pull/1185/files#diff-584caab787ffa39e459a947a816ba2e4R630